modules/rules_appimage/1.7.5/presubmit.yml

```diff
@@ -0,0 +1,8 @@
+matrix:
+  platform: [macos, ubuntu2204]
+  bazel: [6.x, 7.x]
+tasks:
+  run_tests:
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    test_targets: [//...]
```

Comment:
Try `@rules_appimage//...` instead of `//...`.

Reply:
```suggestion
    test_targets: [@rules_appimage//...]
```